.circleci/config.yml

```diff
@@ -134,9 +134,9 @@ commands:
           name: Install python dependencies
           command: python3 -m pip install requests
       - run:
-          name: Verify that a changelog entry is present in the PR description
+          name: Verify that a changelog entry is present in CHANGELOG.md
           command: |
-            MBX_CI_GITHUB_TOKEN=$(./mbx-ci github reader token)
+            MBX_CI_GITHUB_TOKEN=$(./mbx-ci github reader public token)
```

Comment:
Is this needed for `reader` token? As far as I know this is required only for `writer` and `issues` ones.

Reply:
It's not, just fixed it while you were commenting!